### PR TITLE
Improved UI

### LINKED ISSUE #4: Add OIDC support
Would let us use our internal SSO to login, similar to room-server implementation (and also benefit from bonuses like passkeys and 2FA)

### PR DESCRIPTION
**Pull request by Humanoidear**

### CODE REVIEW

File: middleware/.DS_Store
Comment (noahpistilli) [inline middleware/.DS_Store:1]: Remove this please.

File: webpanel/login.go [hunk 1]
```diff
@@ -1,46 +1,98 @@
...
+			OAuth2Token *oauth2.Token
+		}{oauth2Token}
+		data, err := json.MarshalIndent(resp, "", "    ")
+		if err != nil {
+			c.HTML(http.StatusInternalServerError, "error.html", gin.H{
+				"Error": err.Error(),
+			})
+			return
+		}
+		c.Writer.Write(data) */
```
Comment (noahpistilli) [inline webpanel/login.go]: Figure out if you are going to use this.

**Review by noahpistilli — COMMENTED** [review]
Please delete the `mc-assets` folder as well.

**Comment by noahpistilli** [comment]
Also resolve the conflicts and squash.

File: webpanel/login.go (hunk 1 above)
Comment (kouacc) [reply webpanel/login.go]: This part is a leftover of the non-OIDC login. I'll remove it

**Comment by kouacc** [comment]
Should be good now. Some testing might be needed for OIDC

**Pull request merged**